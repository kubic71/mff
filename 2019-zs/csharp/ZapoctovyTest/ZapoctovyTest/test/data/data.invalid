Object
A : A
B : A
C
D
D : C