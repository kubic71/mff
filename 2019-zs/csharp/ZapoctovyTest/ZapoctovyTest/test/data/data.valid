Base
Middle : Base
Derived : Middle
D : Middle
E : Base
F