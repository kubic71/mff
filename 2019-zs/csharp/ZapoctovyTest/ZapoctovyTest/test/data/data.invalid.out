1: Duplicate type Object. First declared on line 0
2: Non existing base type A.
3: Non existing base type A.
6: Duplicate type D. First declared on line 5
4 errors. Stopping.
